jenkins/jenkins2-ha-agents.yaml

```diff
@@ -24,6 +24,7 @@ Metadata:
       - ParentSSHBastionStack
       - ParentAuthProxyStack
       - ParentAlertStack
+      - ParentS3Stack
```

Comment:
The parameter name is very generic. What about using `ParentS3LogsStack` instead?

Reply:
renamed to `ParentS3StackAccessLog`